app/setup_handlers.go

```diff
@@ -42,6 +43,8 @@ func setUpgradeHandler(app *ElysApp) {
 				}
 				consensusParams.Evidence.MaxAgeNumBlocks = NewMaxAgeNumBlocks
```

Comment:
I think the previous changes went with last upgrade, we can remove it.

Reply:
this was supposed to be merged too, will update it